hs_core/templates/pages/my-resources.html

```diff
@@ -34,7 +34,7 @@ <h2 class="page-title">Resources</h2>
                     <th>Title</th>
                     {# <th><i class="text-warning fa fa-star"></i></th> #}
                     <th>Type</th>
-                    <th>Owner</th>
+                    <th>First author</th>
```

Comment:
Dave asks me to change the resource listing page to only show the first author since there could be multiple owners, and it will clutter the resource list page if we list all owners. But for resource landing page, we want to show all owners, and all authors.